script/Ciel/Day 14/14_01B_02_CIEL14B_3_2.txt

```diff
@@ -241,17 +250,17 @@ I can't breathe. I can't move my legs.
 [sha:5d31e40716451df2972dd76198041210e1cad568]{
 -- Page 240, Offset 40132.
 -- ちょうどアルクェイドの手の陰になって、あの竜巻からは守られるだろう。
-She's right on the other side of Arcueid's hand, so she'll be protected from this vortex.
+She's on the other side of Arcueid's hand, in the lee, so she should be safe from this vortex.
 }
 [sha:e104c3586fc33506390c6f54f68f83ecb1ba5ccf]{
 -- Page 240, Offset 40133.
 -- 俺はこのまま、自らの愚かさの代償を払うだけ。
-Right now, I'm the only one who will pay the price for my own foolishness.
+I'm the only one who will pay the price for my foolishness.
```

Comment:
Not entirely sure if this is the intended reading. The above sort of makes one think that, but the construction of the sentence is more like

>Now all that's left for me is to pay the price for my own foolishness.